SU-HKKU/cardinal_service_profile
# CSD-299 August 5 QA Fixes

# READY FOR REVIEW

# Summary
-  August 5 QA Fixes

# Needed By (Date)
- When does this need to be merged by?

# Urgency
- How critical is this PR?

# Steps to Test

1. Pull branch
2. Clear cache
3. Check that the following tweaks have been implemented — see [CSD-299](https://stanfordits.atlassian.net/browse/CSD-299), [CSD-201](https://stanfordits.atlassian.net/browse/CSD-291), and [CSD-305](https://stanfordits.atlassian.net/browse/CSD-305) for specific list.

# Associated Issues and/or People
- [CSD-299](https://stanfordits.atlassian.net/browse/CSD-299)
- [CSD-201](https://stanfordits.atlassian.net/browse/CSD-291)
- [CSD-305](https://stanfordits.atlassian.net/browse/CSD-305)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
